source/installation/config.html.md

```diff
@@ -20,7 +20,8 @@ Here is a sample of what you should put inside.
 # The address of the database
 # (replace USER, PASSWORD, PORT and DATABASE_NAME with your values)
 #
-# If you are using SQlite, use the path of the database file (`plume.db` for instance)
+# If you are using SQlite, use the full path of the database file with forward slashes (`plume.db` for instance)
+#DATABASE_URL=C:/path/to/Plume/plume.db
```

Comment:
Sorry, but this may be confusing as most people will probably install Plume on a Linux server, where this path will be invalid. Maybe add another example with a Linux path, and explain which path is for which OS?

Reply:
Agreed. I will update.